stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -856,6 +959,24 @@ public func _graph<State : _TensorArrayProtocolEnhanced,
   }
 }
 
+
+public func _tffunc<State : _TensorArrayProtocolEnhanced,
```

Comment:
is `_tffunc` expected to be a public API eventually? If so please add some doc.

Reply:
Yes, documented it.